omnisharp.json

```diff
@@ -1,6 +0,0 @@
-{
-    "dotnet": {
-        "projects": "src/*/project.json;test/*/project.json;scripts/*/project.json",
```

Comment:
To be honest, I think "projects" was a hold-out from the days of DNX-support in OmniSharp. When that was dropped, this no longer worked.